services/README.adoc

```diff
@@ -53,35 +55,48 @@ service will be receiving semi-arbitrary blobs of JSON from instances.
 [source]
 ----
     +------------+                           +----------------+
-    | Channel    |                           | Connection     |
+    | channels   |                           | connections    |
```

Comment:
Are those tables? I feel it should be the contrary: make the others singular isn't it? It's what I've always seen and learned for RDBMS tables at least.

Reply:
I'm not sure I understand this comment, is the problem with the diagram, or that table names are plurals?

I'm not sure I see the point in debating the merits of an extra character on table names :smile: